### ISSUE TITLE
Remove Pandas warning on bar chart

### ISSUE THREAD

**Issue by ericmtsai** [ElectionDataAnalysis/election_data_analysis#33]
Running the Analyzer.bar() function produces this warning:
```
/usr/local/lib/python3.7/site-packages/election_data_analysis-0.1-py3.7.egg/election_data_analysis/analyze/__init__.py:685: SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead
```

This does not affect the output but looks ugly. Fix this.

**Comment by ericmtsai** [comment]
Closed by #144
